packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -24,6 +24,10 @@
   outline-offset: 4px;
 }
 
+.usa-checkbox__input:checked + [class*="__label"]::before {
+  background-size: 1.3rem;
+}
```

Comment:
Why do we need to add a USWDS override here in the web component? Wouldn't any necessary styling for the v3 checkbox just come directly from the USWDS module?

Reply:
This was the fix for the check icon size. This is not within the scope of the PR. If overriding is something we want to avoid I can undo this change and create a ticket.

Reply:
Ah, I see. So what I think needs to happen here is that you need to add the USWDS style to the formations overrides stylesheet to convert the rem to px.

This should be converted to pixels because the base font in Formation is different than the base font for USWDS:

![Screenshot 2023-04-21 at 11 27 09 AM](https://user-images.githubusercontent.com/872479/233687650-c7c96103-37ba-43ab-bde5-ddb6790f3bd7.png)

Try adding this to the `global/_formation_overrides.scss` stylesheet (and remove your override in the component):

```css
.usa-checkbox__input:checked + [class*="__label"]::before {
    background-size: rem-override(0.75rem);
}
```

Reply:
Sure, and can I update the `0.75rem` to `1.3rem` ?

Reply:
You should leave it at 0.75rem and let the function `rem-override` convert it to the correct size pixel in `_formation_overrides.scss`